### PR TITLE
Add CloudFoundry CLI Dockerfile

### PR DESCRIPTION
**Pull request by fwilhe**

### CODE REVIEW

File: cf-cli/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    useradd piper --uid 1000 --gid 1000 --shell /bin/bash --create-home && \
+    curl -L --silent "https://cli.run.pivotal.io/stable?release=linux64-binary&source=github" | tar -zx -C /usr/local/bin && \
+    cf --version
+
+USER piper
+WORKDIR /home/piper
+
+RUN cf add-plugin-repo CF-Community https://plugins.cloudfoundry.org \
+    && cf install-plugin blue-green-deploy -f -r CF-Community \
+    && wget --quiet "https://github.com/cloudfoundry-incubator/multiapps-cli-plugin/releases/download/v2.0.7/mta_plugin_linux_amd64" \
```
Comment (daniel-kurzynski) [inline cf-cli/Dockerfile]: v2.0.10 is already available

File: README.md [hunk 2]
```diff
@@ -75,10 +76,10 @@ before opening a pull request.
 
 Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved.
 This file is licensed under the Apache Software License, v. 2 except as noted
-otherwise in the [LICENSE file][license].
+otherwise in the [LICENSE file](https://github.com/SAP/devops-docker-images/blob/master/LICENSE).
```
Comment (1084565) [inline README.md]: You fixed the alias below -- is the change in this line here still required?

Comment (fwilhe) [reply README.md]: I was not aware of this alias feature, so I guess you're right. Thanks, will fix it.

**Comment by fwilhe** [comment]
The dockerfile was taken without change (except for updating the mta plugin) from the s4sdk repo. Please review :)

**Pull request merged**